app/locale/zh-HANS.coffee

```diff
@@ -2213,15 +2213,15 @@ module.exports = nativeDescription: "简体中文", englishDescription: "Chinese
     favorite_prefix: "最喜爱的语言是 "
     favorite_postfix: "。"
     not_member_of_clans: "还不是任何一个部落里的成员。"
-#    certificate_view: "view certificate"
-#    certificate_click_to_view: "click to view certificate"
-#    certificate_course_incomplete: "course incomplete"
-#    certificate_of_completion: "Certificate of Completion"
-#    certificate_endorsed_by: "Endorsed by"
-#    certificate_stats: "Course Stats"
-#    certificate_lines_of: "lines of"
-#    certificate_levels_completed: "levels completed"
-#    certificate_for: "For"
+    certificate_view: "查看证书"
+    certificate_click_to_view: "点击查看证书"
+    certificate_course_incomplete: "课程未完成"
+    certificate_of_completion: "课程完成证书"
+    certificate_endorsed_by: "颁发人 "
```

Comment:
Is the space intentional at the end of the string?

Reply:
yes, that's fine